README.md

```diff
@@ -30,7 +30,10 @@ If you're considering etcd for production use, please see: [production-ready.md]
 
 The latest release and setup instructions are available at [GitHub][github-release].
 
+If you want to develop on latest etcd, clone the repo and checkout the [branch][branch] to work on.
```

Comment:
All development occurs on the master branch which includes new features and bug fixes. Bug fixes should target the master branch and ported to the appropriate release branch as described in the [branch management](./Documentation/branch_management.md) guide.